Mage/src/main/java/mage/abilities/ActivatedAbilityImpl.java

```diff
@@ -180,13 +181,13 @@ public ActivationStatus canActivate(UUID playerId, Game game) {
         // timing check
         //20091005 - 602.5d/602.5e
         boolean asInstant;
-        ApprovingObject approvingObject = game.getContinuousEffects()
+        Set<ApprovingObject> approvingObjects = game.getContinuousEffects()
                 .asThough(sourceId,
                         AsThoughEffectType.ACTIVATE_AS_INSTANT,
                         this,
                         controllerId,
                         game);
-        asInstant = approvingObject != null;
+        asInstant = !approvingObjects.isEmpty();
         asInstant |= (timing == TimingRule.INSTANT);
```

Comment:
I guess it was always this way, but weird to have this separated out instead of part of a single assignment